### PR TITLE
TR2 Test Fix | LPS-174557 | master

### PR DESCRIPTION
**Pull request by luanakarla**
Ticket:
https://issues.liferay.com/browse/LPS-174557

### CODE REVIEW

**Comment by luanakarla** [comment]
ci:test:sf

**Review by nikki-pru — COMMENTED** [review]
1. Por favor verificar Given, When, Then.
2. It's okay to use `And` Example: `Given - and - When - and - Then`
3. Remove ignore if the story is implemented

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase [hunk 1]
```diff
@@ -16,90 +21,65 @@ definition {
...
+		Testray.testrayApisTearDown();
 
-			Refresh();
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
 		}
+	}
 
-		task ("Go to Testray 2 and change the pagination") {
+	@description = "Automation ID: LPS-163753 | Test Summary: Verify as an administrator I can add another Cases in the Add Another button"
```
[the remark this answers is not in the document]
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase:59]: Done.

[the remark this answers is not in the document]
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase:59]: Done. Thanks!

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase [hunk 2]
```diff
@@ -16,90 +21,65 @@ definition {
...
+	@priority = 3
+	test AdminCanAddAnotherCase {
+		task ("Then I go to the 'Cases' page") {
```
[the remark this answers is not in the document]
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase:62]: Done. Thanks!

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase [hunk 3]
```diff
@@ -109,149 +89,175 @@ definition {
...
-				steps = "Auto Test Steps");
+		task ("Given that I create a new 'Case' via API") {
+			JSONTestray.addCases(
+				caseName = ${caseName},
+				caseType = ${caseType},
+				componentName = ${componentName},
+				projectName = ${projectName});
+		}
+
+		task ("Then I go to the 'Cases' page") {
```
[the remark this answers is not in the document]
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase:109]: Done. Thanks!

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase [hunk 4]
```diff
@@ -109,149 +89,175 @@ definition {
...
```
[the remark this answers is not in the document]
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase:140]: Done. Thanks!

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase [hunk 5]
```diff
@@ -109,149 +89,175 @@ definition {
...
 
-			Testray.assertItemName(text = "Cases_1 Case");
+			Testray.assertItemName(text = ${caseName});
 		}
 
 		task ("Then can delete this case and it no longer appears in the list") {
-			TestrayContextMenu.clickContextMenuProjects(name = "Cases_1 Case");
+			TestrayContextMenu.clickContextMenuProjects(name = ${caseName});
 
 			TestrayContextMenu.selectItemFromProjectsContextMenu(item = "Delete");
```
[the remark this answers is not in the document]
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase:202]: Done. Thanks!

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase [hunk 6]
```diff
@@ -109,149 +89,175 @@ definition {
...
+				key_checkboxOption = ${caseName},
 				locator1 = "Checkbox#SPECIFIC_CHECKBOX_IN_TABLE");
 
 			Button.click(button = "Select Cases");
 
-			Testray.assertItemName(text = "Cases_1 Case");
+			Testray.assertItemName(text = ${caseName});
 		}
 
 		task ("Then can delete this case and it no longer appears in the list") {
```
[the remark this answers is not in the document]
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase:198]: Done. Thanks!

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase [hunk 7]
```diff
@@ -109,149 +89,175 @@ definition {
...
-				steps = "Auto Test Steps");
+		task ("Given that I create a new 'Case' via API") {
+			JSONTestray.addCases(
+				caseName = ${caseName},
+				caseType = ${caseType},
+				componentName = ${componentName},
+				projectName = ${projectName});
+		}
 
+		task ("Then I go to the 'Cases' page") {
```
[the remark this answers is not in the document]
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase:218]: Done. Thanks!

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase [hunk 8]
```diff
@@ -109,149 +89,175 @@ definition {
...
-				key_caseComponent = "Components_1 Project 2");
+				caseComponent = "Component 1",
+				caseName = "Case Edited",
+				casePriority = 3);
 		}
 	}
 
-	@description = "This is a test for LPS-156916 - Verify if as an Administrator I am able to Export Cases from another Project"
+	@description = "Automation ID: LPS-156916 | Test Summary: Verify if as an Administrator I am able to Export Cases from another Project"
 	@ignore = "Test Stub"
```
[the remark this answers is not in the document]
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase:261]: Done. Thanks!

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase [hunk 9]
```diff
@@ -260,75 +266,89 @@ definition {
...
 
+		task ("When the user goes to cases and link a requirements") {
 			TestrayCases.linkCasesRequirement(
 				buttonText = "Select Requirements",
 				linkText = "Link Requirements",
-				requirementName = "ATF_Requirements Summary");
+				requirementName = "Requirement Summary");
 		}
 
 		task ("Then the requirements should be displayed on Case page") {
```
[the remark this answers is not in the document]
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase:309]: Done. Thanks!

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase [hunk 10]
```diff
@@ -260,75 +266,89 @@ definition {
...
+			JSONTestray.addRequirement(
+				componentName = ${componentName},
+				linkTitle = "ATF Requirement Link",
+				projectName = ${projectName},
+				requirementName = "Requirement Summary");
 		}
 
-		task ("And Add a new Requirement") {
-			Testray.projectTabNavigator(navTabName = "Requirements");
+		task ("Then I go to the 'Requirement' page") {
```
[the remark this answers is not in the document]
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase:286]: Done. Thanks!

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by nikki-pru** [comment]
ci:test:site-initializer-testray:nocompile

**Comment by nikki-pru** [comment]
ci:stop

**Comment by nikki-pru** [comment]
ci:forward

**Pull request closed without merging**